package.json

```diff
@@ -21,7 +21,9 @@
     "serve": "cd app && yarn serve",
     "serve:win": "cd app && yarn serve:win",
     "lint:global": "cd api && yarn lint --fix && cd .. && cd app && yarn lint --fix && cd .. && cd extension && yarn lint:fix && cd ..",
-    "release": "git stash && git fetch --all && git checkout origin/develop -B develop && git pull && git checkout origin/release -B release && git pull && git merge origin/master && git merge origin/develop && git push && git stash pop"
+    "release:api": "git stash && git fetch --all && git checkout origin/develop -B develop && git pull && git checkout origin/release-api -B release-api && git pull && git merge origin/master && git merge origin/develop && git push && git stash pop",
```

Comment:
soooo much code repetition

Reply:
true true

Reply:
but I am not sure how to avoid it... Need to investigate how to use that param in the script

Reply:
There is a way to do this declaring a variable in package.json, but in Windows needs a different syntax:
https://stackoverflow.com/questions/43705195/how-can-i-use-variables-in-package-json

I think it is better to leave it like this. What do you think?

Reply:
Great! That was it. Much better like this, thank you :pray: 

I leave the script in the root folder for now and we can create a scripts folder in the future, if we need another script.

Tried it and works